core/amber/src/main/python/core/runnables/network_sender.py

```diff
@@ -81,4 +81,7 @@ def _send_control(
             ControlInvocation or ReturnInvocation.
         """
         python_control_message = PythonControlMessage(tag=to, payload=control_payload)
-        self._proxy_client.call_action("control", bytes(python_control_message))
+        returned_credits = int.from_bytes(
+            self._proxy_client.call_action("control", bytes(python_control_message)),
+            byteorder="big",
```

Comment:
I remember when you send it the order you used `little`, why use `big` here? aren't they supposed to be aligned?

Also, what is behavior if we send from different machines with different native byte orders? I assume the byte order can enforce the parsing of the bytes?